### ISSUE TITLE
Error when loading ARM templates with BOM

### ISSUE THREAD

**Issue by kamil-mrzyglod** [ARMataTeam/ARMata#23]
When a template is loaded and it contains BOM, following error occurs:
```
Uncaught SyntaxError: Unexpected token ﻿ in JSON at position 0
    at JSON.parse (<anonymous>)
    at Home.render (bundle.js:902)
    at Home.render (bundle.js:37116)
    at eval (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:796)
    at measureLifeCyclePerf (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:75)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:795)
    at ReactCompositeComponentWrapper._renderValidatedComponent (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:822)
    at ReactCompositeComponentWrapper._updateRenderedComponent (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:746)
    at ReactCompositeComponentWrapper._performComponentUpdate (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:724)
    at ReactCompositeComponentWrapper.updateComponent (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:645)
```

**Comment by kamil-mrzyglod** [comment]
Check this https://gist.github.com/pbakondy/f5045eff725193dad9c7

**Comment by kamil-mrzyglod** [comment]
Fixed in #30
